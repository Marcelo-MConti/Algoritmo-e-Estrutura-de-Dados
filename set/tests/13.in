1
200 0
300 299 298 297 291 289 288 286 285 284 283 282 279 278 277 276 274 272 271 270 269 267 266 265 264 263 262 261 259 258 257 256 255 253 252 250 249 248 246 245 244 243 239 237 235 234 233 231 229 227 225 221 219 218 217 216 213 210 209 208 206 205 203 202 200 198 197 196 195 193 190 189 188 187 186 184 182 181 180 177 176 175 174 173 172 171 169 168 167 165 164 162 160 158 155 154 152 150 148 147 146 145 141 139 136 135 134 133 132 131 129 128 127 126 123 122 120 119 117 116 114 112 111 109 108 107 106 105 103 102 101 99 98 97 94 93 92 90 88 87 86 85 83 82 80 79 78 76 75 72 71 69 65 64 62 61 60 59 58 57 55 54 53 52 51 50 49 48 47 45 44 43 42 41 39 38 37 33 32 31 30 29 27 25 24 22 20 18 17 16 15 13 10 8 7 6 5 3 1 0

4
199
