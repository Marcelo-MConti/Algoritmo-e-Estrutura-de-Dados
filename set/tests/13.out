0, 1, 3, 5, 6, 7, 8, 10, 13, 15, 16, 17, 18, 20, 22, 24, 25, 27, 29, 30, 31, 32, 33, 37, 38, 39, 41, 42, 43, 44, 45, 47, 48, 49, 50, 51, 52, 53, 54, 55, 57, 58, 59, 60, 61, 62, 64, 65, 69, 71, 72, 75, 76, 78, 79, 80, 82, 83, 85, 86, 87, 88, 90, 92, 93, 94, 97, 98, 99, 101, 102, 103, 105, 106, 107, 108, 109, 111, 112, 114, 116, 117, 119, 120, 122, 123, 126, 127, 128, 129, 131, 132, 133, 134, 135, 136, 139, 141, 145, 146, 147, 148, 150, 152, 154, 155, 158, 160, 162, 164, 165, 167, 168, 169, 171, 172, 173, 174, 175, 176, 177, 180, 181, 182, 184, 186, 187, 188, 189, 190, 193, 195, 196, 197, 198, 200, 202, 203, 205, 206, 208, 209, 210, 213, 216, 217, 218, 219, 221, 225, 227, 229, 231, 233, 234, 235, 237, 239, 243, 244, 245, 246, 248, 249, 250, 252, 253, 255, 256, 257, 258, 259, 261, 262, 263, 264, 265, 266, 267, 269, 270, 271, 272, 274, 276, 277, 278, 279, 282, 283, 284, 285, 286, 288, 289, 291, 297, 298, 299, 300, 
